### ISSUE TITLE
virtualbox portable 4.3.28 "network config sample has stop workin" in loop..

### ISSUE THREAD

**Issue by p060477** [vboxme/Portable-VirtualBox#29]
hi!

 

i'm running  vbox latest portable 4.3.28 and if i choose network-bridge i Always get at the start :

"network config sample has stop workin" 

then vbox starts and works with my network -bridge opt- 

but if i esc vbox and then i re enter i got the same pop up a/m

BUT then vbox do not find my network so is not able to start..

i esc again and re entering i got Always the same pop up -network config sample has stop work"

but the vbox works well..and so on...

 

to sum up :

 

the first time i start vbox i get the pop up error but vbox works

if i esc then

the second time same error but vbox do not work

i esc again

third time....vbox works

fourth  NO

five  YES

and so on....

 

what have i to do to solve..??

 

this happens only with win 8.1 as host

with xp as host no

 

i ask on MS forum but the admin reply that

is a matter of the dev of the sw....

 

could you pls help me!!

 

thxs so much in adv!

 

cheers!!

 

:)

**Comment by p060477** [comment]
this is the error pop up:

Nome evento problema:	APPCRASH
  Nome applicazione:	snetcfg_x64.exe
  Versione applicazione:	5.2.3790.0
  Timestamp applicazione:	436618ee
  Nome modulo con errori:	msvcrt.dll
  Versione modulo con errori:	7.0.9600.17415
  Timestamp modulo con errori:	545055fe
  Codice eccezione:	c0000005
  Offset eccezione:	000000000000254d
  Versione SO:	6.3.9600.2.0.0.768.101
  ID impostazioni locali:	1040
  Informazioni aggiuntive 1:	a69c
  Ulteriori informazioni 2:	a69c85ff728b20887852ba2fb6a6c70c
  Ulteriori informazioni 3:	dd06
  Ulteriori informazioni 4:	dd06640a3ddbcaf1f5c14bbfd879ff24

hope this help to solve this nightmare...

;) :(

**Comment by p060477** [comment]
why there is no reply here...
i hope you want to help also a not skill newbie as me..

**Comment by p060477** [comment]
really do not understand why no one has a reply here...

**Comment by Megachip** [comment]
see issue #6
